save-backend/src/main/kotlin/org/cqfn/save/backend/service/OrganizationService.kt

```diff
@@ -0,0 +1,24 @@
+package org.cqfn.save.backend.service
+
+import org.cqfn.save.backend.repository.OrganizationRepository
+import org.springframework.stereotype.Service
+
+/**
+ * Service for organization
+ *
+ * @property organizationRepository
+ */
+@Service
+class OrganizationService(private val organizationRepository: OrganizationRepository) {
+    /**
+     * @param organizationId
+     * @return organization by id
+     */
+    fun getOrganizationById(organizationId: Long) = organizationRepository.findById(organizationId).get()
```

Comment:
Why one of this method called `get`, while another `find`? May be we can stick to one style?

Reply:
Fixed